File: gwanghoon/boj_1978.py
import math

num=int(input())
data=list(map(int,input().split()))
num_list=list(range(2,1001)) 
cnt=0

for i in range(2,math.ceil(math.sqrt(1000))):   #ceil = 올림
    for j in num_list:
        if j/i == 1:   
            pass
        elif j%i==0: 
            num_list.remove(j)  
for k in data:
    if k in num_list:
        cnt+=1
print(cnt)  

File: cocorig/BOJ/boj_1978.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : './input.txt';
let input = fs.readFileSync(filePath).toString().trim().split('\n')

const list =  input[1].split(' ').map(Number);//[ 1, 3, 5, 7 ]

function isPrime(num){

    if(num === 1) return false; // 1은 소수가 아님
    for(let i = 2; i <= parseInt(Math.sqrt(num)); i++){
      if(num  % i === 0 ) return false;
      // 다른 수와 나누어 떨어진다면 소수가 아님
    }
    return true;
  }



console.log(list.filter(item => isPrime(item)).length);


